pkg/variablefinders/requirements.go

```diff
@@ -105,6 +65,54 @@ func FindRequirements(g gitclient.Interface, jxClient jxc.Interface, ns, dir, ow
 	return req, nil
 }
 
+func GetSettings(g gitclient.Interface, jxClient jxc.Interface, ns, dir, owner, repo string) (*jxcore.Settings, string, error) {
```

Comment:
can you add a godoc comment about what the function does, similar to what `FindRequirements` has.

Reply:
OK